query/src/index.ts

```diff
@@ -1,18 +1,22 @@
-export { injectQueryClient, provideQueryClient } from './lib/query-client';
-export { injectQuery } from './lib/query';
-export { injectMutation } from './lib/mutation';
+export { injectInfiniteQuery } from './lib/infinite-query';
 export { injectIsFetching } from './lib/is-fetching';
 export { injectIsMutating } from './lib/is-mutating';
-export { injectInfiniteQuery } from './lib/infinite-query';
-
+export { injectMutation } from './lib/mutation';
+export { injectQuery } from './lib/query';
 export {
-  toPromise,
-  createSuccessObserverResult,
-  createPendingObserverResult,
-} from './lib/utils';
-export * from './lib/operators';
+  QueryClient,
```

Comment:
Is there a reason we need to expose QueryClient?

Reply:
No, for some reasons i tought was necessary for testing. Removed.